cartography/intel/gcp/cloudcdn.py

```diff
@@ -342,6 +338,17 @@ def get_global_url_maps(compute: Resource, project_id: str) -> List[Dict]:
         else:
             raise
 
+@timeit
+def transform_global_url_maps(url_maps: List[Dict], project_id: str) -> List[Dict]:
+    global_url_maps = []
+    for url_map in url_maps:
+        url_map['region'] = 'global'
+        url_map['type'] = 'global'
+        url_map['consolelink'] = gcp_console_link.get_console_link(project_id=project_id, resource_name='cdn')
```

Comment:
it's not supported in cloudconsolelinks project yet.

Reply:
I have added the the consolelink implementeation for new resources and raised a PR